content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+  - User triggers, e.g. a user visits a specific repository for the first time, if recurring, a user should feel added value each time they see the message.    
+- **Timebox**
+  - Maximum number of days a coordinated campaign will run across all targeted users.
+  - Maximum number of times a user will see an in-product message until dismissed.
+
+## Avoid traffic collisions
+Avoid showing too many alerts at the same time. Collisions are awkward and undesireable. Consider what other alerts, teaching bubbles, or banners may be active on the page before launching your onboarding experience.
+
+![docsImage -  Avoid traffic](https://user-images.githubusercontent.com/47256527/88720302-7fdea600-d0d9-11ea-9dcb-0b4100114d7f.png)
+
+## Create efficient sequences
+Sequence tasks in a logical way to increase onboarding success. Clearly define when someone has started setup and successfully completed your flow. Treat your feature onboarding as a story or a journey with a:
+
+- Beginning
+- Middle 
+- End
+
+## Draw attention to your feature
+Use teaching bubbles, alert banners, the feature preview tray, or announcement modals to draw attention to your feature.
+
+- ✅ Use empty states or alert banners to call attention to a feature if users will discover your feature in order to start a task
+- ✅ Use teaching bubbles or other easily dismissable elements if users will discover your feature while they are completing another task
+
+## Begin a setup flow
+Use [empty states](https://primer.style/design/ui-patterns/empty-states), [teaching bubbles](https://docs.microsoft.com/en-us/javascript/api/office-ui-fabric-react/iteachingbubbleprops?view=office-ui-fabric-react-latest), or a [CTA on a marketing page](https://github.com/features/actions) on GitHub to begin a setup flow.
+
+![blank state](https://user-images.githubusercontent.com/47256527/88720353-97b62a00-d0d9-11ea-93d6-951c9da55d9c.png)
+
+![Actions example 1](https://user-images.githubusercontent.com/47256527/88720395-a56baf80-d0d9-11ea-9bd6-e98df7a9f27e.png)
+
+
+## Guide your user through their task
+Use a [form](https://primer.style/css/components/forms), [checklist](https://primer.style/css/components/forms), or other UI elements to help users complete their task. 
+
+![Checklist example](https://user-images.githubusercontent.com/47256527/88720479-c7fdc880-d0d9-11ea-9ac1-76b9defafe79.png)
+
+## Celebrate the finish
+Use a toast, flash alert, or other message to confirm the user has completed their task. Message should be short and brief.
+
+![Do](https://user-images.githubusercontent.com/47256527/88720500-d3e98a80-d0d9-11ea-8943-43e6e04431b3.png)
+
+ 
+# Teaching bubble
+Teaching bubbles are a special [popover]() that calls attention to a feature in a specific part of the page. Generally, features highlighted by teaching bubbles should be used to educate the user and enriching to the user’s task at hand. Consider the page the user is currently viewing and what jobs-to-be-done they may be performing.
```

Comment:
"Generally, features highlighted by teaching bubbles should be used to educate the user and enriching to the user’s task at hand." - it feels like the bit "features highlighted by" doesn't fit the meaning of the sentence, and the sentence is describing the actual teaching bubbles, so: "Generally, teaching bubbles should be used to educate the user and enrich the task at hand."

Also, can we find a replacement for "jobs-to-be-done"? task?